src/track/track.h

```diff
@@ -290,6 +290,9 @@ class Track : public QObject {
     void setMainCuePosition(mixxx::audio::FramePos position);
     /// Shift all cues by a constant offset
     void shiftCuePositionsMillis(mixxx::audio::FrameDiff_t milliseconds);
+    /// Shift beatgrid by a constant offset
+    void shiftBeatsMillis(double milliseconds);
```

Comment:
Not trying to get greedy, but while you're at it, could you fix the signature of the sibling method above as well? Framediff for milliseconds looks definitely wrong.

Reply:
Sure, I spotted that, too, but didn't want to do too much refactoring until we came to a conclusion regarding the beats+cues implementation.